### PR TITLE
layout_stars

### PR DESCRIPTION
**Pull request by DmitryVolchokFE**
https://dmitryvolchokfe.github.io/layout_stars/
https://dmitryvolchokfe.github.io/layout_stars/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
```
Comment (volodymyr-soltys97) [inline src/style.css:30]: Change the selector `.stars--4 :nth-child(-n + 4)` to `.stars--4 .stars__star:nth-child(-n + 4)` to target the correct child elements.

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job
But let's make your code more sophisticated
Let's remove the right indentation for the last elements
image

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1.stars__star:nth-child(5),
+.stars--2.stars__star:nth-child(5),
+.stars--3.stars__star:nth-child(5),
+.stars--4.stars__star:nth-child(5),
+.stars--5.stars__star:nth-child(5) {
+  margin-right: 0;
+}
```
Comment (nazarmatsevych) [inline src/style.css:41]:
```suggestion
.stars__star:last-child {
  margin-right: 0;
}
```

Should work as well

**Review by nazarmatsevych — APPROVED** [review]

**Comment by DmitryVolchokFE** [comment]
image
You need to confirm the task again

**Review by nazarmatsevych — APPROVED** [review]
No problem but i think it doesnt help to fix this problem, you have to create new form and new pull request to get this task done

**Pull request closed without merging**